lerna.json

```diff
@@ -1,8 +1,7 @@
 {
-  "lerna": "2.0.0-beta.32",
-  "packages": [
-    "."
-  ],
+  "lerna": "2.5.1",
+  "npmClient": "yarn",
+  "useWorkspaces": true,
   "changelog": {
```

Comment:
I'm not sure, but I think we might need to still duplicate the `packages` config in lerna.json for the benefit of `lerna-changelog`, the consumer of this config....